docs/GUIDE_FOR_KOGITO_DEVS.md

```diff
@@ -0,0 +1,352 @@
+# Introduction
+This is a list of tasks/activities that are useful or necessary 
+to know while working on the Kogito operator. 
+This document is written in an order that is recommended to 
+be worked through chronologically as a new Kogito operator developer.
+
+# Installing Kogito Dependencies
+## CRC
+CRC (CodeReady Containers) allows you to have a locally 
+running OpenShift container on your laptop which is 
+essential for testing when making changes to the operator. 
+[This article](https://developers.redhat.com/blog/2019/09/05/red-hat-openshift-4-on-your-laptop-introducing-red-hat-codeready-containers/) 
+gives you a good overview of the product as well where to 
+download it and ensure that it works. You will need to 
+[create a Red Hat account](https://www.redhat.com/wapps/ugc/register.html) 
+to download CRC.
+
+To get the command to login as an admin, you can use `crc 
+console --credentials`.
+
+## Operator
+The operator requires some dependencies in order to run. The 
+following dependencies are copied from the [operator's README](https://github.com/kiegroup/kogito-operator#requirements). 
+Please ensure that they are up-to-date and still match. Following 
+are the specific installation commands/links for Fedora:
+
+### Single Command
+Operator Courier, Go, Golint, Python 3.x, Cekit:
```

Comment:
I would recommend installing CeKit using virtualenv.

Reply:
Shouldn't this fine since the [CEKit installation docs](https://docs.cekit.io/en/latest/handbook/installation/instructions.html) mention installing it with `dnf` for Fedora? They also mentioning using `virtualenv` for other Linux distributions, which I did mention in the guide. But, I recommended using `venv` since that's what the [Python docs](https://docs.python.org/3/tutorial/venv.html) mention and apparently [`virtualenv`](https://virtualenv.pypa.io/en/latest/) was partially integrated into `venv`.